azure-pipelines/builds/ci.yml

```diff
@@ -17,68 +17,6 @@ stages:
 - stage: build
   displayName: Build
   jobs:
-  - template: /eng/common/templates/jobs/jobs.yml
+  - template: /eng/common/templates/job/build-from-source.yml
```

Comment:
Put this in `/eng/templates/...` to avoid blocking arcade auto-updates? Later we can uptake and delete this copy.

Reply:
I can do that.